packages/main/src/Link.ts

```diff
@@ -248,6 +283,9 @@ class Link extends UI5Element implements ITabbable {
 			&& this._isCrossOrigin();
 
 		this._rel = needsNoReferrer ? "noreferrer noopener" : undefined;
+
+		// Icon only link is not allowed
+		this._ensureNoIconOnlyLink();
```

Comment:
You can't throw errors from components for incomplete API. For example, imagine the icon is hard-coded for a link, but the text property is bound to a model which hasn't resolved yet, and it is set 100ms later when React updates the link. However, you already threw an error which could break the whole application.

There are the following possibilities:
 - Just render the 2 icons (do nothing to handle the error scenario)
 - Do not render anything (so keep the checks, but use the getter in the template to wrap the whole output in an `if` statement).

I would personally prefer the first one. Please research how native HTML elements behave in similar scenarios that don't make sense.